active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+| CanDragTabs | Gets or sets a value that indicates whether tabs in the collection can be dragged. Default is true. |
+| CanReorderTabs | Gets or sets a value that indicates whether tabs in the collection can be reordered through user interaction. Default is true. |
+| IsAddTabButtonVisible | Determines if the plus button appears to the right of the tab strip |
+| TabItemsSource | Gets or sets an object source used to generate the tabs. |
+| TabItems | Gets the collection used to generate the tabs. |
+| ItemTemplate | Gets or sets the DataTemplate used to display each item. |
+| ItemTemplateSelector | Gets or sets a reference to a custom DataTemplateSelector logic class. The DataTemplateSelector referenced by this property returns a template to apply to items. |
+| SelectedIndex | Gets or sets the index of the selected item. |
+| SelectedItem | Gets or sets the selected item. |
+| TabStripHeader | Content to the left of the tab strip. |
+| TabStripFooter | Content to the right of the tab strip. |
+| TabViewItemWidthMode | Specifies how the tabs should be sized. Values are {Actual, Equal}. Default is Actual. |
+
+| Event | Description |
+|---|---|
+| AddTabButtonClick | Fires when the + button is clicked. |
+| SelectionChanged | Occurs when the currently selected tab changes. |
+| TabClosing | Fires when a tab is about to be closed. Can be cancelled to prevent closure. |
+| TabDraggedOutside | Fires when a Tab is dragged outside of the Tab bar. |
+
+| Method | Description |
+|---|---|
+| DependencyObject ContainerFromItem(object item) | Returns the generated container for the specified item. |
+| DependencyObject ContainerFromIndex(Int32 index) | Returns the generated container for the specified index. |
+
+### TabViewItem properties and events
+
+| Property | Description |
+|:-------- |:----------- |
+| Content | The main content that appears in the tab. |
+| Header | The content that appears inside the tab itself.  |
+| HeaderTemplate | Template for the header object. |
+| Icon | Icon for the tab. |
+| IsCloseable | Determines if the tab shows a close button. Default is "True". |
+
+| Event | Description |
+|---|---|
+| TabClosing | Fires when a tab's close button is clicked. |
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```
+namespace MU_XC_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+enum TabViewItemWidthMode
+{
+    Actual = 0,
+    Equal = 1,
+};
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+runtimeclass TabViewTabClosingEventArgs
+{
+    Object Item { get; };
+
+    Boolean Cancel { get; set; };
+}
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.Control
+{
+    TabView();
+
+    [MUX_DEFAULT_VALUE("winrt::TabViewItemWidthMode::Actual")]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    TabViewItemWidthMode TabViewItemWidthMode{ get; set; };
+
+    // TODO: Remove
```

Comment:
Are these TODOs still relevant?

Reply:
These TODOs generally mark deltas between what we have implemented and the "end goal". In this case, we have an API in code that we should remove since it's not desirable for the final control.

Reply:
Recommend: remove TODOs from the spec